src/components/LMPost/LMPostHeader/index.tsx

```diff
@@ -0,0 +1,226 @@
+import {View, Text, StyleSheet, Image, TouchableOpacity} from 'react-native';
+import React, {useState} from 'react';
+import STYLES from '../../../constants/constants';
+import layout from '../../../utils/layout';
+import LMProfilePicture from '../../../base/LMProfilePicture';
+import {LMPostHeaderProps} from './types';
+import {timeStamp} from '../../../utils';
+import LMPostMenu from '../LMPostMenu';
+import LMText from '../../../base/LMText';
+
+const LMPostHeader = ({
+  user,
+  onProfileTap,
+  profileTextStyle,
+  profileTextViewStyle,
+  imageViewStyle,
+  imageSize,
+  titleText,
+  createdAt,
+  isEdited,
+  showMemberStateLabel,
+  memberState,
+  memberStateTextStyle,
+  memberStateViewStyle,
+  postHeaderViewStyle,
+  pinIcon,
+  isPinned,
+  menuItems,
```

Comment:
Rather than taking custom styling variables in the "molecules", let's take an "atom" object as the object to be displayed. This is how the true power of this analogy will come into play. If we keep three `TextStyle` type variables then we will be asking for them in any component that would be using it.